lib/qarakusiscene.py

```diff
@@ -1,73 +1,11 @@
-from manim import UP, DOWN, LEFT, RIGHT
+from manim import UP, LEFT
 from manim import VGroup
-from manim import MathTex
-from manim import YELLOW
-from manim import Scene, Write
-from manim import UpdateFromAlphaFunc
-
-
-class Task(VGroup):
-    """Կամյական խնդրի ձևակերպում"""
-
-    def __init__(
-        self,
-        number: MathTex,
-        text: VGroup,
-        **kwargs
-    ):
-        super().__init__()
-
-        self.number = number.set_color(YELLOW)
-        self.text = text.arrange(DOWN, aligned_edge=LEFT)
-
-        self.add(self.number, self.text.align_to(self.number, UP))
-        self.arrange(RIGHT, aligned_edge=UP)
-
-
-class QarakusiScene(Scene):
```

Comment:
I'm not sure (@zidder or @narek-mkrtumyan would say) but you are not to remove QarakusiScene class

Reply:
I think it's okay to delete it until we know what we want from the scene to do